### DISCUSSION TITLE
Checkbox: onChange handler type

### DISCUSSION THREAD

**Discussion by DaniilNaletov** [heroui-inc/heroui#1442] [Help] [answered]
Hi! I got a problem with `<Checkbox />` onChange handler. In documentation it's described as a default input handler: `ChangeEvent<HTMLInputElement>`, but in reality it's showed up as a `FormEvent<HTMLLabelElement>`, so I got an error, when trying to use default `ChangeEvent<HTMLInputElement>` handler for `<Checkbox />`. The type definition comes from `react/index.d.ts`
Help please)

<img width="997" alt="image" src="https://github.com/nextui-org/nextui/assets/5238005/c77e8bdc-2fe4-41f3-bf19-08f64495e4f6">
<img width="829" alt="image" src="https://github.com/nextui-org/nextui/assets/5238005/8f40fe06-cb50-4024-9f75-06efbb27a617">

**Comment by jrgarciadev** [comment, marked as answer]
Hey @DaniilNaletov update to the latest NextUI version

**Reply by DaniilNaletov** [reply]
I've just updated to version `2.0.24` and It works! Thank you 🙏
